client/extensions/woocommerce/state/sites/settings/stripe-connect-account/test/reducer.js

```diff
@@ -523,4 +527,362 @@ describe( 'reducer', () => {
+								lastName: 'Foo',
+								logo: '',
+								oauthUrl: 'https://connect.stripe.com/oauth/authorize',
+							},
+						},
+					},
+					456: {
+						settings: {
+							stripeConnectAccount: {
+								connectedUserID: 'acct_14qyt6Alijdnw0EA',
+								displayName: 'Foo Bar',
+								email: 'foo@bar.com',
+								error: '',
+								firstName: 'Foo',
+								isActivated: false,
+								isDeauthorizing: true,
+								isOAuthInitializing: false,
+								isRequesting: false,
+								lastName: 'Bar',
+								logo: '',
+								oauthUrl: '',
+							},
+						},
+					},
+				},
+				action
+			);
+			expect( newState[ 123 ].settings.stripeConnectAccount.isOAuthInitializing ).to.eql( false );
+			expect( newState[ 123 ].settings.stripeConnectAccount.oauthUrl ).to.eql(
+				'https://connect.stripe.com/oauth/authorize'
+			);
+			expect( newState[ 456 ].settings.stripeConnectAccount.isOAuthInitializing ).to.eql( false );
+			expect( newState[ 456 ].settings.stripeConnectAccount.oauthUrl ).to.eql( '' );
+		} );
+	} );
+
+	describe( 'connectAccountOAuthInitComplete w/ Error', () => {
+		test( 'should set the error in state', () => {
+			const action = {
+				type: WOOCOMMERCE_SETTINGS_STRIPE_CONNECT_ACCOUNT_OAUTH_INIT_COMPLETE,
+				siteId: 123,
+				error: 'My error message',
+			};
+			const newState = stripeConnectAccountReducer( undefined, action );
+			expect( newState.error ).to.eql( 'My error message' );
+		} );
+
+		test( 'should leave other sites state unchanged', () => {
+			const action = {
+				type: WOOCOMMERCE_SETTINGS_STRIPE_CONNECT_ACCOUNT_OAUTH_INIT_COMPLETE,
+				siteId: 123,
+				error: 'My error message',
+			};
+			const newState = sitesReducer(
+				{
+					123: {
+						settings: {
+							stripeConnectAccount: {
+								connectedUserID: 'acct_14qyt6Alijdnw0EA',
+								displayName: 'Foo Bar',
+								email: 'foo@bar.com',
+								error: '',
+								firstName: 'Foo',
+								isActivated: false,
+								isDeauthorizing: false,
+								isOAuthInitializing: true,
+								isRequesting: false,
+								lastName: 'Bar',
+								logo: '',
+							},
+						},
+					},
+					456: {
+						settings: {
+							stripeConnectAccount: {
+								connectedUserID: 'acct_14qyt6Alijdnw0EA',
+								displayName: 'Foo Bar',
+								email: 'foo@bar.com',
+								error: '',
+								firstName: 'Foo',
+								isActivated: false,
+								isDeauthorizing: false,
+								isOAuthInitializing: true,
+								isRequesting: false,
+								lastName: 'Bar',
+								logo: '',
+							},
+						},
+					},
+				},
+				action
+			);
+			expect( newState[ 123 ].settings.stripeConnectAccount.error ).to.eql( 'My error message' );
+			expect( newState[ 123 ].settings.stripeConnectAccount.isOAuthInitializing ).to.eql( false );
+			expect( newState[ 456 ].settings.stripeConnectAccount.error ).to.eql( '' );
+			expect( newState[ 456 ].settings.stripeConnectAccount.isOAuthInitializing ).to.eql( true );
+		} );
+	} );
+
+	/////
```

Comment:
Looks like this should be removed?

Reply:
LOL. Yep. Fixed in https://github.com/Automattic/wp-calypso/commit/77666c56bca1bb9e208934bea5f4769733bc2a8b